install_manually.md

```diff
@@ -8,7 +8,7 @@ Once you have access to your server, either directly or by SSH, you can install
 
 1. Install git
 ```bash
-sudo apt-get install git
+sudo apt-get install git dialog
```

Comment:
Why do you propose `dialog` package?
Can't we install it from install script or YunoHost debian dependency?
Installing `git` is only to clone `install_script` repository.